.github/workflows/ci.yml

```diff
@@ -611,8 +610,6 @@ jobs:
           - suite-5
           - suite-azure
           - suite-delta-lake-databricks
-        jdk:
-          - 11
```

Comment:
Does that mean that most product tests were not in fact run on JDK17?

Reply:
They were on my branch so I was able to fix most of the problems.